scripts/modules/b2b-account/account-hierarchy.js

```diff
@@ -0,0 +1,140 @@
+define(["modules/jquery-mozu", 'modules/api', "underscore", "hyprlive", "modules/backbone-mozu", "hyprlivecontext", 'modules/models-customer'], function ($, api, _, Hypr, Backbone, HyprLiveContext, CustomerModels) {
+
+    var AccountHierarchyView = Backbone.MozuView.extend({
+        templateName: "modules/b2b-account/account-hierarchy/account-hierarchy",
+        render: function () {
+            var self = this;
+            Backbone.MozuView.prototype.render.apply(this, arguments);
+            self.fixMargins();
+        },
+        initialize: function () {
+            Backbone.MozuView.prototype.initialize.apply(this, arguments);
+        },
+        fixMargins: function () {
+            $(document).ready(function () {
+                //close menu if we click on outside of menu
+                $(document).on("click", function (event) {
+                    if (!$(event.target).closest(".dropdown-content").length && !$(event.target).closest(".three-dots").length) {
+                        $(".account-hierarchy .dropdown-content").removeClass("active");
+                    }
+                });
+
+                //fix margins
+                $(".node:not(:has(.caret))").css("margin-left", "-10px");
```

Comment:
Minor but ideally you would add/remove a class from the HTML element instead of adding CSS directly. I'd recommend adding a `.fix-margin` class or something with the rule and then use the jQuery method `addClass(name)`.